spec/models/ability_spec.rb

```diff
@@ -210,7 +208,7 @@
       it{ should_not be_able_to(:manage, my_conference.tickets.first) }
       it{ should_not be_able_to(:manage, conference_public.tickets.first) }
 
-      it{ should_not be_able_to(:manage, my_registration) }
+      it{ should be_able_to(:manage, my_registration) }
```

Comment:
It is obviously wrong that I used two `user` in same scope.

> How come you changed the test?

It was failing.. I really don't think I understand `ability_specs` :confused: You will either have to take time to explain it to me or you can send a new PR using my first commit. 
I would prefer first, may be we can work on refactoring it and there is other work on authorization we have discussed. At least we should divide it in files(on basis of roles) so that we don't have to scroll 200 lines to see the definition of each variable.